src/linker/System.Diagnostics.CodeAnalysis/LinkerRemovableAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Diagnostics.CodeAnalysis
+{
+	/// <summary>
+	/// This attribute name will be the name hardcoded in linker which will remove all 
+	/// attribute usages but not the attribute definition
+	/// </summary>
+	[AttributeUsage (
+		AttributeTargets.Class, Inherited = false)]
+	public sealed class LinkerRemovableAttribute : Attribute
```

Comment:
The name should be very descriptive: `RemoveAttributeInstancesAttribute` - to match the actual name in the XML.